src/components/SelectionListWithSections/index.tsx

```diff
@@ -107,6 +107,9 @@ function SelectionListWithSections<TItem extends ListItem>({onScroll, shouldHide
             isRowMultilineSupported
             shouldDisableHoverStyle={shouldDisableHoverStyle}
             setShouldDisableHoverStyle={setShouldDisableHoverStyle}
+            // We only allow the prop to pass on Native (for scroll offset calculation)
+            // web should be false by default since there are no issues on web
+            isPercentageMode={false}
```

Comment:
This prop is not a generic prop, and it has no relation to this component. Can we replace this with a generic one?

Reply:
I think for now let it be as I am getting pings to ship it fast.